src/openapi.yaml

```diff
@@ -3853,10 +3853,10 @@ paths:
               $ref: '#/components/headers/X-Is-Form-Designer'
             Content-Language:
               $ref: '#/components/headers/Content-Language'
-  /api/v2/registration/plugins/objects-api/catalogi:
+  /api/v2/registration/plugins/objects-api/catalogues:
```

Comment:
Just to double check: these endpoints are only used for the OF admin, right?

Reply:
indeed - only `/api/v2/public/` is public API :)